components/src/dynamo/vllm/handlers.py

```
@@ -576,6 +533,39 @@ async def scale_elastic_ep(self, body: dict) -> dict:
+            # ray.nodes() uses the GCS gRPC channel directly (no dashboard process
+            # needed) and returns the same information. Imported lazily so ray is not
+            # required at module load time (absent in non-elastic-EP deployments).
+            #
+            # Format mapping:
+            #   list_nodes() → objects with .node_ip and .node_id
+            #   ray.nodes()  → dicts with "NodeManagerAddress" and "NodeID"
+            import ray
+            import ray.util.state as _ray_util_state
+
+            class _NodeInfo:
+                __slots__ = ("node_ip", "node_id")
```

Ruff (0.15.6)

[warning] `_NodeInfo.__slots__` is not sorted

Apply a natural sort to `_NodeInfo.__slots__`

(RUF023)